reports.sh

```diff
@@ -97,36 +104,42 @@ function deployAllureResults() {
     extractAllureResults
     generateAllureReports
     echo "LOG1"
-    url=$(deployToNetlify "allure-report")
+    url="$(deployToNetlify "allure-report")"
     echo "LOG2"
     sendComment "$(aboutNetlify ${url})"
 }
 
 function downloadAllureResults() {
     urlExistence=false
-    for url in $(collectRelevantComments "${TRAVIS_BUILD_NUMBER}")
+    echo "TRAVIS_BUILD_NUMBER: ${TRAVIS_BUILD_NUMBER}"
+    for urlKey in $(collectRelevantComments "${TRAVIS_BUILD_NUMBER}")
     do
         urlExistence=true
-        echo "Found: ${url}"
-        fileName="$(echo "${url}"| awk -F/ '{print $5}')"
-        curl ${url} --output ${fileName}
+        echo "Found: ${urlKey}"
+        # TODO: $4 for file.io, #5 for transfer.sh, add an IF
+        #fileName="$(echo "${url}.tar.gz"| awk -F/ '{print $5}')"
+        fileName="${urlKey}.tar.gz"
+        tmpResult="$(curl https://file.io/${urlKey} --output ${fileName})"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [tmpResult appears unused. Verify use (or export if used externally).](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)

Reply:
codacy will be fixed later